### PR TITLE
New content

### PR DESCRIPTION
**Pull request by crisgon**
- GraphQl Posts in portuguese
- Git and Github course in portuguese

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +188,8 @@ Achou algum recurso interessante? Escreveu um post? Contribua com o projeto!
 ### GraphQL
 
 - [How to GraphQL](https://www.howtographql.com/basics/0-introduction/) :us: :keyboard: :free:
+- [GraphQL —A flexibilidade do formato de dados](https://medium.com/trainingcenter/graphql-os-dados-no-formato-requisitado-964f5555d467) :brazil: :page_facing_up: :free:
```
Comment (lucianomlima) [inline README.md]: Add a space between dash and A, and you're ready to go!
```suggestion
- [GraphQL — A flexibilidade do formato de dados](https://medium.com/trainingcenter/graphql-os-dados-no-formato-requisitado-964f5555d467) :brazil: :page_facing_up: :free:
```

**Review by lucianomlima — CHANGES_REQUESTED** [review]

**Comment by crisgon** [comment]
Feito!

File: README.md [hunk 2]
```diff
@@ -188,6 +188,8 @@ Achou algum recurso interessante? Escreveu um post? Contribua com o projeto!
 ### GraphQL
 
 - [How to GraphQL](https://www.howtographql.com/basics/0-introduction/) :us: :keyboard: :free:
+- [GraphQL — A flexibilidade do formato de dados](https://medium.com/trainingcenter/graphql-os-dados-no-formato-requisitado-964f5555d467) :brazil: :page_facing_up: :free:
+- [GraphQL  —  Aprendendo na prática](https://medium.com/trainingcenter/graphql-aprendendo-na-pr%C3%A1tica-569a6866065b) :brazil: :page_facing_up: :free:
```
Comment (lucianomlima) [inline README.md]: You put a extra space here 😆
```suggestion
- [GraphQL —  Aprendendo na prática](https://medium.com/trainingcenter/graphql-aprendendo-na-pr%C3%A1tica-569a6866065b) :brazil: :page_facing_up: :free:
```

**Review by lucianomlima — APPROVED** [review]
Nice!

**Pull request merged**